$NetBSD: patch-ai,v 1.4 2014/05/14 02:28:13 rodent Exp $

Correct a misspelling.  Keep the hunks at their current release offsets so
GNV patch does not enter its case-losing backup path on OpenVMS.

--- languages/english/english.3.orig
+++ languages/english/english.3
@@ -5044,6 +5044,7 @@
 dirndl
 disaccredit
 disafforest
+disappear
 disbranch
 disbud
 discalced
@@ -5077,7 +5078,6 @@
 dispersoid
 displayable
 disproven
-dissappear
 disseminule
 dissentious
 dissepiment
